bnb/getBNBData.sh

```diff
@@ -1,16 +0,0 @@
-#!/bin/bash
-
```

Comment:
This script is also mentioned in the README and should therefore been updated there as well.

Reply:
In fact it is mentioned in a section of INSTALL.md which could be updated with more detailed instructions on how to set up a CRON job for it.

Reply:
Refer to 94c0b51